armtemplates/template.bicep

```
@@ -0,0 +1,272 @@
+param virtualMachines_iagora_name string = 'iagora'
+param publicIPAddresses_iagora_ip_name string = 'iagora-ip'
+param virtualNetworks_iagora_vnet_name string = 'iagora-vnet'
+param networkInterfaces_iagora731_z1_name string = 'iagora731_z1'
+param networkSecurityGroups_iagora_nsg_name string = 'iagora-nsg'
+param sshPublicKey string 
+param vmSize string = 'Standard_B2s' 
+
+//NSG ressourc
+resource networkSecurityGroups 'Microsoft.Network/networkSecurityGroups@2023-11-01' = {
+  name: networkSecurityGroups_iagora_nsg_name
+  location: 'francecentral'
+  properties: {
+    securityRules: [
+      {
+        name: 'SSH'
+        
+        properties: {
+          protocol: 'TCP'
+          sourcePortRange: '*'
+          destinationPortRange: '22'
+          sourceAddressPrefix: '*'
+          destinationAddressPrefix: '*'
+          access: 'Allow'
+          priority: 300
+          direction: 'Inbound'
+          sourcePortRanges: []
+          destinationPortRanges: []
+          sourceAddressPrefixes: []
+          destinationAddressPrefixes: []
+        }
+      }
+      {
+        name: 'HTTP'
+        properties: {
+          protocol: 'TCP'
+          sourcePortRange: '*'
+          destinationPortRange: '80'
+          sourceAddressPrefix: '*'
+          destinationAddressPrefix: '*'
+          access: 'Allow'
+          priority: 320
+          direction: 'Inbound'
+          sourcePortRanges: []
+          destinationPortRanges: []
+          sourceAddressPrefixes: []
+          destinationAddressPrefixes: []
+        }
+      }
+      {
+        name: 'HTTPS'
+       // id: networkSecurityGroups_iagora_nsg_name_HTTPS.id
+        //type: 'Microsoft.Network/networkSecurityGroups/securityRules'
+        properties: {
+          protocol: 'TCP'
+          sourcePortRange: '*'
+          destinationPortRange: '443'
+          sourceAddressPrefix: '*'
+          destinationAddressPrefix: '*'
+          access: 'Allow'
+          priority: 340
+          direction: 'Inbound'
+          sourcePortRanges: []
+          destinationPortRanges: []
+          sourceAddressPrefixes: []
+          destinationAddressPrefixes: []
+        }
+      }
+    ]
+  }
+}
+
+
+// PUBLIC IP ADDRESS RESSOURCE
```

```suggestion
// PUBLIC IP ADDRESS RESOURCE
```
Oups, this type is present in multiple comments...